modules/social_features/social_event/modules/social_event_invite/src/Form/EnrollInviteEmailForm.php

```diff
@@ -178,6 +203,26 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#limit_validation_errors' => [],
     ];
 
+    // We should prevent invite enrollments if social_event_max_enroll is
+    // enabled and there are no left spots.
+    if ($this->moduleHandler->moduleExists('social_event_max_enroll')) {
+      $event_max_enroll_service = $this->eventMaxEnrollService;
+
+      if (
+        $node instanceof NodeInterface &&
+        $event_max_enroll_service->isEnabled($node)
+      ) {
+        // If there are no spots left, disable button and add the button title
+        // with appropriate notice.
+        if ($event_max_enroll_service->getEnrollmentsLeft($node) === 0) {
```

Comment:
This if statement can be merged with the above one.

Reply:
Done